scenarioo-server/src/main/java/org/scenarioo/dao/configuration/ConfigurationDaoImpl.java

```diff
@@ -34,30 +33,31 @@
 	
 	private static final Logger LOGGER = Logger.getLogger(ConfigurationDaoImpl.class);
 	
-	private static final String USER_HOME_BASE_DIRECTORY = ".scenarioo";
 	private static final String CONFIG_FILE_NAME = "config.xml";
-	private static final String DEFAULT_CONFIG_PATH = CONFIG_FILE_NAME;
-	
-	private String configurationDirectory = null;
-	private String configurationFilename = null;
+
+	private final String configurationDirectory;
 	
-	public ConfigurationDaoImpl(final String configurationDirectory, final String configurationFilename) {
+	public ConfigurationDaoImpl(final String configurationDirectory) {
 		this.configurationDirectory = configurationDirectory;
-		this.configurationFilename = configurationFilename;
 	}
 	
 	@Override
 	public Configuration loadConfiguration() {
 		File configFile = getFileSystemConfigFile();
-		if (configFile == null || !configFile.exists()) {
+		if (!configFile.exists()) {
 			LOGGER.warn("  file " + configFile + " does not exist --> loading default config.xml from classpath");
-			configFile = getClasspathConfigFile();
-		} else {
-			LOGGER.info("  loading configuration from file: " + configFile);
+			return ScenarioDocuXMLFileUtil.unmarshal(Configuration.class, getDefaultConfigFile());
 		}
+
+		LOGGER.info("  loading configuration from file: " + configFile);
 		return ScenarioDocuXMLFileUtil.unmarshal(Configuration.class, configFile);
```

Comment:
@mi-we 
I know the else block is not necessary here - but ...
nevertheless I would recommend to put one - to make it more explicit (easier to read) that only one or the other is done.

Reply:
I changed it on my other branch (just merge my PR #636)